wasm/v8/module.go

```diff
@@ -46,7 +48,10 @@ func (mod *V8Module) ExecuteNewCall(
 
 	// Runs all scripts (will be changed depending on files needed), probably going to merge all that are needed. This if makes sure we load our needed scripts ONLY on the first call
 	if cachedInstance == nil {
+
 		_ = injectStoreFunction(inst.ctx, call)
+		_ = injectClockFunction(inst.ctx, call.Clock)
```

Comment:
Either check the return value or omit it completely. Also, the full code to create an instance should be move to its own function I think, at least soon as we are going to add more and more stuff.